### PR TITLE
feat(commit.ts) : Get the current branch after remote selected for 'git push'

### PR DESCRIPTION
**Pull request by jessicakuijer**
I've added a little improvement to have the perfect workflow when user select its remote to push code.

### CODE REVIEW

File: src/commands/commit.ts [hunk 1]
```diff
@@ -78,39 +71,32 @@ ${chalk.grey('——————————————————')}`
...
-        const { stdout } = await execa('git', ['push', selectedRemote]);
-        pushSpinner.stop(
-          `${chalk.green(
-            '✔'
-          )} successfully pushed all commits to ${selectedRemote}`
-        );
-
-        if (stdout) outro(stdout);
-      } else outro(`${chalk.gray('✖')} process cancelled`);
+    const isPushConfirmedByUser = await confirm({
```
Comment (di-sukharev) [inline src/commands/commit.ts:77]: prettier issue

**Review by di-sukharev — COMMENTED** [review]
please merge latest dev into your branch and change base branch to dev to merge to

File: src/commands/commit.ts [hunk 2]
```diff
@@ -185,6 +172,8 @@ export async function commit(
       .join('\n')}`
   );
 
+  await generateCommitMessageFromGitDiff(staged.diff);
```
[the remark this answers is not in the document]
Comment (jessicakuijer) [reply src/commands/commit.ts]: that's right, sorry. I didn't fetch/pull the actual code. And everything works now without any changes.

**Comment by jessicakuijer** [comment]
I've uninstalled and installed opencommit again with actual codes. I close this PR, it's not necessary.

**Pull request closed without merging**